3. Installation/3. Docker Containers/systemd/README.md

```diff
@@ -4,22 +4,103 @@ Deploy [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) to Linux that ru
 
 ## Prerequisite
 
-You need to have [docker](https://docs.docker.com/linux/started/) and [docker-compose](http://docs.docker.com/compose/) installed.
+You need to have [docker](https://docs.docker.com/linux/started/) installed.
 
 ## How to run Rocket.Chat on systemd.
 
+1. Create a docker network `docker network create rocketchat_default`
+2. Create two service files: mongo.service and rocketchat.service
+3. Enable the service files
+4. Run the mongo.service
+5. Create the mongo-init-replica to enable opolog
+6. Run the rocketchat.service
+
+If you reboot the server, the services will start automatically in the right order.
+
+### The docker-container for the mongo-init-replica will remain in an exited state. Do NOT remove the container `mongo-init-replica`!
```

Comment:
@thiesschneider since you start the container and connect to the other mongo container, no data will be saved in this containers volume.  So completely disposable, you are only linking for the purposes of allowing the containers to talk, not inheriting it's data volumes.  

Also since you are actually mounting the mongo container to a local disk directory you are even doubly safe.